src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -976,4 +982,39 @@ public void dbSNPError() {
         //make sure it doesn't throw an error
         runCommandLine(args);
     }
+
+    @Test
+    public void testNoReadsOutputAsNoCall() {
```

Comment:
The input data for this test comes from a GVCF produced from an empty region of one of the NA12878 test bams and a GVCF that was edited to have a variant so we can force that position to be output. (I should probably put this test description in the code.)

Reply:
👍